### ISSUE TITLE
FNX-5418 ⁃ [Bug] The tab order is potentially confusing for Chrome users

### ISSUE THREAD

**Issue by Lululalu** [mozilla-mobile/fenix#2048]
## Steps to reproduce
Open several new tabs and links in new tabs

### Expected behavior
Chrome opens new tabs under the current tab, always from the top to the bottom. Users are extremely used to this behavior, so if Fenix does the same, the transition between browsers may more comfortable and easier.

### Actual behavior
Fenix opens new tabs the opposite way, from the bottom to the top, and it's confusing and uncomfortable for long time chrome users (myself included).

### Device information
* Android device: 8.0
* Fenix version: 1.0.1917

**Comment by cadeyrn** [comment]
It's not a bug. The tab order was intentionally changed in #538 (and I am very happy about that!). Not everything Chrome does is the best.

**Comment by Lululalu** [comment]
@cadeyrn I understand that not everything Chrome does is the best, but as users are used to such tab order, I bet several users will feel that the Fenix's tab order is confusing.. At least, there should be a setting where user can choose the tab order. I, personally, would choose the older tab order, it feels better to me.

**Comment by yoasif** [comment]
I don't think that the tab listings of Fenix vs. Chrome are really analogous, given that Fenix uses tiles and Chrome uses cards. I think that if Fenix used cards, the Chrome order makes a lot more sense.

With tiles, it is more of an open question - I personally wouldn't care which order it was in if #1827 was fixed. FWIW, Fennec shows the newest tabs at the bottom, unlike current Fenix.

**Comment by vesta0** [comment]
Thanks @Lululalu, will make sure to test this during user research.
